e2e/tests/playwright-tests/config.ts

```diff
@@ -0,0 +1,24 @@
+export const userAuthData = {
```

Comment:
😩

Can you start a thread on #qa about migrating this stuff to a secret and rotating the creds? I see *some* of these are already checked in to other GitHub repos in virtru, but I'd like to avoid checking in credentials as much as possible.

Reply:
I forgot this is a public repo. We'll have to rotate these credentials. I think we can store the regenerated ones in vault or something?

Reply:
Removed, I'll use this approach https://github.com/virtru/automated-test-scripts-v3/blob/develop/.buildkite/hooks/pre-command#L19